src/Linux/Avalonia.LinuxFramebuffer/LinuxFramebufferPlatform.cs

```diff
@@ -152,5 +157,79 @@ public static int StartLinuxDirect<T>(this T builder, string[] args, IOutputBack
         builder.Instance.Run(lifetime.Token);
         return lifetime.ExitCode;
     }
+
+    public static DrmOutput CreateDrmOutput<T>(this T builder, string path = null, bool connectorsForceProbe = false,
```

Comment:
This is implemented for currently splash screen hack where we create the DrmOuput manually to draw a Lottie animation and then pass it as IOutputBackend to StartLinuxDirect

Reply:
Why does it need the application builder? 
Also if it's a public static method, we might as well move it to `DrmOutput.CreateWithAutoDetect`

Reply:
I agree